protos/dcs/tts/v0/tts.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+package dcs.tts.v0;
+import "dcs/common/v0/common.proto";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+
+service TtsService {
+  rpc Transmit(TransmitRequest) returns (TransmitResponse) {}
+}
+
+message TransmitRequest {
+  // The text that is synthesized to speech and transmitted to SRS. Supports SSML tags (you should
+  // not wrap the text in the root `<speak>` tag though).
+  string text_ssml = 1;
+
+  // The plain text without any transformations made to it for the purpose of getting it spoken out
+  // as desired (no SSML tags, no FOUR NINER instead of 49, ...). Even though this field is
+  // optional, please consider providing it as it can be used to display the spoken text to players
+  // with hearing impairments.
+  optional string text_plain = 2;
+
+  // The radio frequency in Hz the transmission is send to. Example: 251000000
+  // for 251.00MHz.
+  uint64 frequency = 3;
+
+  // Name of the SRS client. Defaults to "DCS-gRPC".
+  optional string name = 4;
```

Comment:
Could we add this to the TTS event so that we can format a message like: `251.000MHz | NameOfSender | Message`. May be useful for scripted missions.

And if we do then update the doc to reflect that it will be the SRS client name _and_ the name displayed in events.

Reply:
Done, also renamed to `srs_client_name` as it seem to unspecific to me looking at it again.